db/migrate/20180226175554_create_cars.rb

```diff
@@ -0,0 +1,20 @@
+class CreateCars < ActiveRecord::Migration[5.1]
+  def change
+    create_table :cars do |t|
+      t.string :make
+      t.string :model
+      t.integer :year
+      t.text :description
+      t.integer :number_seats
+      t.boolean :convertible
+      t.integer :price_day
+      t.boolean :driver_left
+      t.boolean :automatic
+      t.boolean :chauffeur
+      t.string :location
+      t.references :owner, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Adding defaults:

https://stackoverflow.com/questions/8627156/adding-default-true-to-boolean-in-existing-rails-column